modyn/supervisor/supervisor.py

```diff
@@ -51,6 +53,9 @@ def __init__(
         if not self.validate_pipeline_config():
             raise ValueError("Invalid pipeline configuration")
 
+        if not is_directory_writable(self.eval_directory):
+            raise ValueError("No permission to write to the evaluation results directory.")
+
```

Comment:
we might also want to check that if evaluation is given, there are no duplicate dataset ids (instead of doing this when making the gRPC call, we just fail early) + there is at least one dataset (non empty lsit) in the input validation

Reply:
Done, see that check the evaluation pipeline config similar to the training config.